### DISCUSSION TITLE
AutoARIMA in-sample prediction

### DISCUSSION THREAD

**Discussion by Srobins7** [alan-turing-institute/sktime#3802] [Q&A]
Hello,

I just had a quick question re AutoARIMA.  In previous versions of sktime (13.2) I was able to generate in-sample predictions for a simple univariate time series by simply using negative forecast horizons e.g., fitted_model.predict(fh=[-1]).

In newer versions of sktime, passing negative numbers to predict fh returns NaN values even if I only use just default parameters for model/the same data set.  Was wondering if the ability to generate in-sample forecasts was for this type of model was deprecated or changed?  I couldn't find any information on this in discussion nor bugs submitted so am thinking it might be that I'm just not doing it correctly.

Thanks for your help!

**Comment by fkiraly** [comment]
@Srobins7, I'm not aware of anything that would have changed.

Can you provide some code with an example that used to work and no longer does not?

May be a bug.

**Reply by Srobins7** [reply]
Sure thanks for your response. This is an example with 14.0 that used to work in 13.2:

`
import pandas as pd
from sktime.forecasting.arima import AutoARIMA

test_data = pd.DataFrame(
data=(np.arange(1,20)*100), 
columns=['test_data'], 
index=pd.PeriodIndex(data=np.arange(2000,2019),freq='Y'))

model = AutoARIMA()
model.fit(test_data)
model.predict(fh=-np.arange(0,6))
`
The output dataframe is all NaN values from this.  The data set I'm working with is small/annual so dummy test_data is representative/normally I would set seasonal=False however it doesn't work even with default params.

**Reply by Srobins7** [reply]
@fkiraly just wanted to follow up on this, did you encounter the same issue I did?  or did I specify things incorrectly?  Would much prefer to use a more recent version of sktime on my current project than revert back to 13.2 just to be able to produce in-sample forecasts.

**Reply by Srobins7** [reply]
An update after trying some different dependency combos - It seems that I had an older version of pmdarima installed (1.8.5) which wasn't working properly with sktime (13.3+).  After updating pmdarima to more recent version (2.0.1) the AutoARIMA in-sample prediction now works.
